polkadot/xcm/pallet-xcm/src/lib.rs

```diff
@@ -429,7 +445,7 @@ pub mod pallet {
 		/// The location is invalid since it already has a subscription from us.
 		AlreadySubscribed,
 		/// Invalid asset for the operation.
-		InvalidAsset,
+		InvalidAsset(AssetTransferError),
```

Comment:
I think nowadays named fields are preferred over anonymous fields in pallet error variants.

Reply:
done